docs/playbook/training-modules/Module-01.md

```diff
@@ -0,0 +1,90 @@
+# What we'll cover in Module 1
+
+In this module you will ingest two datasets into the Data Platform from a Google Sheet. This will involve creating an AWS [Glue](https://lbhackney-it.github.io/Data-Platform-Playbook/glossary/#glue) job using [Terraform](https://lbhackney-it.github.io/Data-Platform-Playbook/glossary/#terraform) as well as [GitHub](https://lbhackney-it.github.io/Data-Platform-Playbook/glossary/#github) for making changes to the Data Platform code repository. Once your changes have been approved by the Data Platform team, you will then crawl the loaded data, to then be able to view and query it in [AWS Athena](https://lbhackney-it.github.io/Data-Platform-Playbook/glossary#athena), a database-like web client.
+
+
+## Step-by-step instructions
+
+Please ensure that [Module 0](https://playbook.hackney.gov.uk/Data-Platform-Playbook/playbook/training-modules/Module-0) is completed before starting this module.
+
+### 1. Preparing the Google Sheet for this exercise
+Make a copy of this Covid vaccination [data](https://docs.google.com/spreadsheets/d/1-ZNoQGu0LGlaKYDBWD8MUo8hqfcnE5YbgCXVz2MUxSw/edit#gid=2146898708) and name the Google Sheet as `covid_vaccination_data` in your own Google Drive. We are going to use the tabs *locations* and *vaccinations*.
+
+
+### 2. Granting the Data Platform access to your new Google sheet
+We have created a `sandbox` department in the Data Platform, and an associated Google service account: `sandbox@dataplatform-stg.iam.gserviceaccount.com`
+- The Data Platform will use this account to connect to your Google sheet. 
+- Use the sharing settings of your sheet and grant “Viewer” permissions to this user.
+- [Detailed steps](https://playbook.hackney.gov.uk/Data-Platform-Playbook/playbook/ingesting-data/google-sheets-import#preparing-a-google-sheet-for-ingestion) are in the Playbook.
+
+
+### 3. Creating two ingestion jobs in Terraform
+Your Google sheet is going to be ingested by a job running in AWS [Glue](https://lbhackney-it.github.io/Data-Platform-Playbook/glossary#glue). You will write a bit of [Terraform](https://lbhackney-it.github.io/Data-Platform-Playbook/glossary/#terraform) that will deploy this job automatically in the AWS environment. You are going to write this code directly in your web browser in GitHub here, adding to the existing `Terraform script 26-google-sheets-imports.tf`.
+
+![Terraform script](../images/edit_terraform_google_sheets.png)
+
+You could also write the job locally within your interactve development environment, but this would remove the ability to test the code due to the way the Data Platform infrastructure is currently set up. Therefore, editing via the GitHub web interface will be easier in this case. 
```

Comment:
Why does writing it locally remove the ability to test the code? (Appreciate you probably copied this from the google doc but was wondering if you know)